src/main/main.scss

```diff
@@ -0,0 +1,111 @@
+@import url("https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap");
+
+//font-size dekstop
+$xxs-font-size-desktop: 24px;
+$xs-font-size-desktop: 30px;
+$s-font-size-desktop: 40px;
+$m-font-size-desktop: 48px;
+$l-font-size-desktop: 65px;
+$xl-font-size-desktop: 144px;
+
+//font-size mobile
+$xxs-font-size-mobile: 16px;
+$xs-font-size-mobile: 20px;
+$s-font-size-mobile: 26px;
+$m-font-size-mobile: 32px;
+$l-font-size-mobile: 42px;
+$xl-font-size-mobile: 92px;
+
+// breakpoints
+$m-breakpoint: 767px;
+$l-breakpoint: 1023px;
+$xl-breakpoint: 1399px;
+//colors
+$white-color: #ffffff;
+$black-light-color: rgba(0, 0, 0, 0.45);
+$black-dark-color: rgba(0, 0, 0, 0.8);
+
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    color: $white-color;
+    font-family: "Lato", sans-serif;
```

Comment:
Move font import to `variables.scss` and save it under a variable, then use a variable here.